connector/protobuf/pom.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
```

Comment:
What about schema-registry support? I can look into that. Let me know @mposdev21, @SandishKumarHN

Reply:
Sure, Raghu.  Go ahead.

Reply:
Sure, I will take that.